coalib/core/Bear.py

```diff
@@ -179,24 +197,11 @@ def maintainers_emails(cls):
                 if cls.MAINTAINERS_EMAILS == set() else
                 cls.MAINTAINERS_EMAILS)
 
-    @enforce_signature
-    def __init__(self, section: Section, file_dict: dict):
+    def __attrs_post_init__(self):
         """
-        Constructs a new bear.
-
-        :param section:
-            The section object where bear settings are contained.
-        :param file_dict:
-            The file-dictionary containing a mapping of filenames to the
-            according file contents.
-        :raises RuntimeError:
-            Raised when bear requirements are not fulfilled.
+        This function is called after all attributes are initialized.
+        It checks for pre-requisites.
```

Comment:
"prerequisite" is a legal word, so I would remove the `-` if you want to keep the docstring ;)